warehouse/manage/views.py

```diff
@@ -2944,6 +2950,19 @@ def remove_organization_project(project, request):
             request.route_path("manage.project.settings", project_name=project.name)
         )
 
+    if (
+        # Check that user has permission to remove projects from organization.
+        (project.organization and request.user not in project.organization.owners)
+        # Check that project has an individual owner.
+        or not project_owners(request, project)
+    ):
+        request.session.flash(
+            "Could not remove project from organization", queue="error"
```

Comment:
Should we add a reason to this message? Perhaps "Could not remove project from organization, you do not have the required permissions". Otherwise people may interpret it as a system error.

Reply:
Added clearer error message in commit [6afafa0](https://github.com/pypi/warehouse/pull/12008/commits/6afafa0479efa813257dafb90dd67a196d35dae1).